fleet/view/js/list_fleet.js

```diff
@@ -0,0 +1,792 @@
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var dir = split_url[2];
+var url_host = window.location.origin + "/" + split_url[1] + "/" + split_url[2];
+
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
+
+console.log(split_host[2]);
+var index = window.location.origin + "/" + split_url[1]
+console.log(split_url[1])
+
+//var datatable_list;
+var list_do = [];
+
+let datatable_list; //variable datatable
+
+let data_list_tbl=[], data_list_def=[], flg_data_ready = 0;
+
+let fav_id = 'tbl_fleet';
+
+let menu = [
+    [1, 'Plate Number', 'plate', 0],
+    [2, 'Truck Type', 't_type', 0],
+    [3, 'Type Of Body' , 't_cat', 0],
+    [4, 'Right Driver', 'r_dri', 0],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400071)
```suggestion
    [4, "Right Driver", 'r_dri', 0],
```